Clojure/Clojure.sublime-syntax

```diff
@@ -255,7 +255,7 @@ contexts:
       scope: punctuation.definition.string.end.clojure
       pop: true
     - match: ''
-      push: scope:source.regexp#base-literal
+      push: scope:source.regexp-common#base-literal
```

Comment:
Maybe I am missing something, but this line doesn't change anything on my box. With or without the change, Clojure syntax tests pass. I've even renamed the shipped "Regular Expressions.sublime-package".

Reply:
It seemed to fail on the CI runner, though...

Reply:
Even runs fine on CI (https://github.com/deathaxe/sublime-packages/tree/debug_CI)

Reply:
My bad, the CI failure I observed was when I had pushed my changes to `Regexp.sublime-syntax` as the "common" syntax, with the Boost specific stuff in a separate definition, in an attempt to get a nicer diff. Sorry for the confusion and misinformation. I'll revert that commit then because it would be a breaking change for Clojure regexps to only highlight common syntax.